lib/galaxy/tools/parameters/validation.py

```diff
@@ -186,7 +186,9 @@ def __init__(self, message, length_min, length_max, negate):
         super().__init__(message, range_min=length_min, range_max=length_max, negate=negate)
 
     def validate(self, value, trans=None):
-        super().validate(len(value), trans)
+        if value is None:
+            raise ValueError("No value provided")
+        super().validate(len(value) if value else 0, trans)
```

Comment:
Do we need this additional check for value being None?

Reply:
Yes, otherwise you're getting a TypeError, see the linked issue.